app/models/businessmatching/BusinessActivities.scala

```diff
@@ -65,27 +67,30 @@ object BusinessActivity {
 
   implicit val activityFormRead = Rule[String, BusinessActivity] {
       case "01" => Valid(AccountancyServices)
-      case "02" => Valid(BillPaymentServices)
-      case "03" => Valid(EstateAgentBusinessService)
-      case "04" => Valid(HighValueDealing)
-      case "05" => Valid(MoneyServiceBusiness)
-      case "06" => Valid(TrustAndCompanyServices)
-      case "07" => Valid(TelephonePaymentService)
+      case "02" => Valid(ArtMarketParticipant)
+      case "03" => Valid(BillPaymentServices)
+      case "04" => Valid(EstateAgentBusinessService)
+      case "05" => Valid(HighValueDealing)
+      case "06" => Valid(MoneyServiceBusiness)
+      case "07" => Valid(TrustAndCompanyServices)
+      case "08" => Valid(TelephonePaymentService)
       case _ => Invalid(Seq((Path \ "businessActivities") -> Seq(ValidationError("error.invalid"))))
   }
 
   implicit val activityFormWrite = Write[BusinessActivity, String] {
       case AccountancyServices => "01"
-      case BillPaymentServices => "02"
-      case EstateAgentBusinessService => "03"
-      case HighValueDealing => "04"
-      case MoneyServiceBusiness => "05"
-      case TrustAndCompanyServices => "06"
-      case TelephonePaymentService => "07"
+      case ArtMarketParticipant => "02"
+      case BillPaymentServices => "03"
+      case EstateAgentBusinessService => "04"
+      case HighValueDealing => "05"
+      case MoneyServiceBusiness => "06"
+      case TrustAndCompanyServices => "07"
+      case TelephonePaymentService => "08"
   }
 
   implicit val jsonActivityReads: Reads[BusinessActivity] = Reads {
     case JsString("01") => JsSuccess(AccountancyServices)
+    case JsString("08") => JsSuccess(ArtMarketParticipant)
```

Comment:
Question - should the numeric value here not match the numeric value in the writes?

Reply:
I updated the indices on the forms and model to maintain the ordering, because the presubmission section was looping through and using the index of the loop to retrieve messages from the message file - I wanted to insert AMP into the correct place in the list.

The existing JSON indices were not modified in order to maintain backwards compatibility with existing user data.

(I later realised that telephone payment and trusts and company services are the wrong way around anyway, so the additional effort probably wasn't strictly necessary...)